app/views/rentals/processing.html.erb

```diff
@@ -3,23 +3,23 @@
 <%= search_form_for @q, url: '/rentals/processing', class: 'form-inline' do |f| %>
   <%= label_tag :item_type_id_in, 'Containing Item Types' %>
   <%= select_tag :item_type_id_in, options_from_collection_for_select(@item_types.order(name: :asc), "id", "name", selected: params[:item_type_id_in]),
-    multiple: true, include_blank: true, class: "selectpicker form-control col-xs-2" %>
+    multiple: true, include_blank: true, class: "selectpicker form-control col-xs-2" %> <br></br>
```

Comment:
Just a note:
`<br>` is one of those tags that doesn't to be explicitly closed because nothing is even allowed to be between the opening and closing tag.
You could replace this with just a `<br>` by itself without the closing tag or if you want to be a bit more concise you can also use `<br />`

Reply:
I tried those originally, but the space between the rows still looked clunky.